### DISCUSSION TITLE
Linux con WSL

### DISCUSSION THREAD

**Discussion by loyolacamil** [dds-jv/foro#338] [Consultas de Arquitectura] [answered]
Buenas! queria consultarles sobre la opcion de descargar linux a traves de wsl, que me den alguna referencia, y si es compatible con la materia.. gracias!
saludos,
Camil

**Comment by gastonprieto** [comment, marked as answer]
buenas @loyolacamil 

la verdad no lo use nunca, debería ser compatible supongo, pero no conozco la configuración del mismo, ni si tiene algún problema
